src/main/java/com/razorpay/PaymentClient.java

```diff
@@ -26,23 +29,19 @@ public Payment capture(String id, JSONObject request) throws RazorpayException {
     return post(String.format(Constants.PAYMENT_CAPTURE, id), request);
   }
 
-  public Refund refund(String id) throws RazorpayException {
```

Comment:
dont remove the older methods. Mark them as deprecated. Add java doc to point to new method